dimscord/constants.nim

```diff
@@ -416,6 +416,63 @@ type
         flNotSet =      0
         flListView =    1
         flGalleryView = 2
+    DispatchEvent* = enum
+      Unknown
```

Comment:
Will be used if the event can't be parsed. Means a bot won't crash when new events are added that are not supported